src/sentry/sentry_metrics/indexer/postgres/postgres_v2.py

```diff
@@ -195,8 +261,17 @@ def reverse_resolve(self, use_case_id: UseCaseKey, org_id: int, id: int) -> Opti
         string: str = obj.string
         return string
 
-    def _table(self, use_case_id: UseCaseKey) -> IndexerTable:
-        return TABLE_MAPPING[use_case_id]
+    def _get_metric_path_key(self, use_case_ids: Collection[UseCaseID]) -> UseCaseKey:
+        if UseCaseID.SESSIONS in use_case_ids:
+            if len(use_case_ids) > 1:
+                raise ValueError(
+                    f"The set of use_case_ids: {use_case_ids} maps to multiple metric path keys"
+                )
```

Comment:
This check shouldn't be inside the `if UseCaseID.SESSIONS in use_case_ids` condition, let's make it the first check of the function

Reply:
Okay, but just to confirm: In the future, there could be multiple generic use case IDs, so it will eventually move into the `if` statement like it is now, right?

Reply:
I think the right thing to check is that all items of the collection map to the same use case key. Checking `len(use_case_ids) == 1` is sufficient for now because only two exist. But I think this check has to change regardless

Reply:
What about something like
```python
def _get_metric_path_key(self, use_case_ids: Collection[UseCaseID]) -> UseCaseKey:
    metrics_paths = {METRIC_PATH_MAPPING[use_case_id] for use_case_id in use_case_ids}
    if len(metrics_paths) > 1:
        raise ValueError(
            f"The set of use_case_ids: {use_case_ids} maps to multiple metric path keys"
        )
    return next(iter(metrics_paths))
```

Reply:
i'm fine with that

Reply:
Fixed, thanks!